docs/user-guide/config-examples.md

```diff
@@ -52,6 +52,7 @@ all of the nixvim options are accessible no matter how you are using it (flake,
 [khaneliman/khanelivim]: https://github.com/khaneliman/khanelivim
 [MikaelFangel/nixvim-config]: https://github.com/MikaelFangel/nixvim-config
 [nicolas-goudry/nixvim-config]: https://github.com/nicolas-goudry/nixvim-config
+[NikolayGalkin/gnvim]: https://github.com/NikolayGalkin/gnvim
```

Comment:
Looks like this is just adding the link but not adding it to the table

Reply:
You are right. Sorry for my stupid mistake.